### PR TITLE
Finished testing for placeship.

### PR DESCRIPTION
**Pull request by kbajno**
As of now we have 98% coverage, woo!

### CODE REVIEW

File: src/main/java/edu/oregonstate/cs361/battleship/Main.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public static void main(String[] args) {
         //This will listen to GET requests to /model and return a clean new model
         get("/model", (req, res) -> newModel());
         //This will listen to POST requests and expects to receive a game model, as well as location to fire to
-        post("/fire/:row/:col", (req, res) -> fireAt(req));
+     //   post("/fire/:row/:col", (req, res) -> fireAt(req));
```
Comment (leian7) [inline src/main/java/edu/oregonstate/cs361/battleship/Main.java:22]: :+1: to be readded in my firing PR

File: src/test/edu/oregonstate/cs361/battleship/MainTest.java [hunk 2]
```diff
@@ -87,14 +101,20 @@ public void testFireAt() {
...
     }
 
-    private TestResponse request(String method, String path) {
+    private TestResponse request(String method, String path, String body) {
         try {
             URL url = new URL("http://localhost:4567" + path);
             HttpURLConnection connection = (HttpURLConnection) url.openConnection();
             connection.setRequestMethod(method);
             connection.setDoOutput(true);
+            if(body != null) {
```
Comment (leian7) [inline src/test/edu/oregonstate/cs361/battleship/MainTest.java:110]: from the canvas discussion? nice

**Review by leian7 — APPROVED** [review]
Thanks for figuring out all the post request stuff :+1:

**Comment by kbajno** [comment]
One thing I noticed is the fire request is commented out in Main.java! Make sure to fix that.

**Pull request merged**